docs/Android.md

```diff
@@ -30,13 +30,13 @@ Swift-to-Java bridging.
 To follow along with this guide, you'll need:
 
 1. A Linux environment capable of building Swift from source, specifically
-   Ubuntu 16.04 or Ubuntu 15.10 (Ubuntu 14.04 has not been tested recently).
-   The stdlib is currently only buildable for Android from a Linux environment.
-   Before attempting to build for Android, please make sure you are able to build
-   for Linux by following the instructions in the Swift project README.
-2. The latest version of the Android NDK (r16 at the time of this writing),
+   Ubuntu 18.04 or Ubuntu 16.04 (Arch Linux has also been tried, but the test
```

Comment:
If it isn’t fully supported, let’s just drop mention of Arch

Reply:
Done.